app/views/games/index.html.erb

```diff
@@ -1,182 +1,51 @@
 <div class="container-fluid">
-  <section id="register">
-    <div class= "row">
+  <section id="games-list">
+    <div class="row">
       <div class="col-sm-5 col-sm-offset-1">
         <%= image_tag 'chess.jpg', class: 'img-responsive feature-pic' %>
       </div>
-      <div class="col-sm-3 col-sm-offset-1 new-account">
-      <%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
-        <%= f.error_notification %>
-        <div class="form-group">
-          <%= f.email_field :email, class: 'form-control', required: true, autofocus: true, placeholder: 'Email Address' %>
-        </div>
-        <br />
-        <div class="form-group">
-          <%= f.password_field :password, class: 'form-control', required: true, hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length), placeholder: 'Password' %>
-        </div>
-        <br />
-        <div class="form-group">
-          <%= f.password_field :password_confirmation, class: 'form-control', required: true, placeholder: 'Confirm Password' %>
-        </div>
-        <br />
-        <div class="form-group">
-          <%= f.button :submit, "Create An Account", class: "btn btn-success form-control" %>
-        </div>
-      <% end %>
+      <div class="col-sm-5">
+        <h2 class="join-headline text-center">Your opponent awaits you! Join the battle!</h2>
+          <h4>Available game<a class="btn btn-success pull-right" href="#" role="button">Join game!</a></h4>
```

Comment:
This would look better & scale for mobile if you instead nest a row div, then put the available game in a col-6 w/ `text-left` class and the Join game button in another col-s w/ `text-right` class